docs/content/en/v2/api/controls/layout.md

```diff
@@ -0,0 +1,29 @@
+---
+id: "v2-layout"
+title: "Layout"
+description: ""
+lead: ""
+date: 2022-06-12T00:00:00+00:00
+lastmod: 2022-06-12T00:00:00+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "controls"
+toc: true
+---
+
+**Inheritance:** [Element]({{< ref "element.md" >}}) -> [NavigableElement]({{< ref "navigableelement.md" >}}) -> [VisualElement]({{< ref "visualelement.md" >}}) -> [View]({{< ref "view.md" >}})
+**Xamarin.Forms documentation:** Layout [API](https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.layout)
+
+## Constructors
+
+This control can't be instantiated on its own. Its properties and events are inherited by its descendants.
+
+## Properties
+
+TBD
```

Comment:
Could you write the properties/events for both Layout and TemplatedView so I can merge this PR, please?

Reply:
For TemplatedView, you can simply remove the `Properties` and `Events` sections.
TemplatedView doesn't have any modifier

Reply:
Updated following your suggestions :)